eZ/Publish/API/Repository/ContentService.php

```diff
@@ -538,4 +539,6 @@ public function newContentMetadataUpdateStruct(): ContentMetadataUpdateStruct;
      * @return \eZ\Publish\API\Repository\Values\Content\ContentUpdateStruct
      */
     public function newContentUpdateStruct(): ContentUpdateStruct;
+
+    public function validate(ValueObject $object, array $context, ?array $fieldIdentifiers = null): array;
```

Comment:
Should we use `$fieldGroupsIdentifiers` instead ? Do we need to go down to individual fields level ?

Reply:
From API point of view I think it is worth going down to individual field level. You will be able to partially validate object programmatically and low level for whatever reason without the need to group fields in advance.